corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -32,11 +34,11 @@ def handle(self, username, **options):
         if couch_user:
             if not isinstance(couch_user, WebUser):
                 raise CommandError('Username already in use by a non-web user')
-            print("✓ User {} exists".format(couch_user.username))
+            logging.info("✓ User {} exists".format(couch_user.username))
```

Comment:
Is it intentional that you're using the root logger here? A more standard convention is to create a logger for each module:
```py
# at top of module under imports
log = logging.getLogger(__name__)

# further below, in class, function, etc.
log.info("✓ User {} exists".format(couch_user.username))
```

Reply:
It certainly doesn't _need_ to be the root logger here if that's what you mean.  As I understand it, the use-case for instantiating a logger is to distinguish where a log originates from in the log messages and to provide log control.  Since this is a management command, the usage is much closer to `print()` in that it's expected that the messages will always be displayed, and their origin will always be clear.  This differs from say, logging a query that happens way down the stack somewhere.

That is, these messages are part of the primary purpose of the code.  It makes sense to me that they be logged this way (`print` would be preferable, actually) - would you disagree?

Reply:
I think it's fine if this management command will only be used via command line. However, a module-specific logger would be better if it is ever invoked programmatically with [`call_command()`](https://docs.djangoproject.com/en/2.2/ref/django-admin/#running-management-commands-from-your-code).

Reply:
Hmm, that's a compelling point
https://github.com/dimagi/commcare-hq/pull/24863